assets/styles/main.css

```diff
@@ -0,0 +1,168 @@
+body {
+  overflow-x: hidden;
+}
+
+main {
+  text-align: left;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.flex-h,
+.flex-v {
+  display: flex;
+}
+
+.flex-v {
+  flex-direction: column;
+}
+
+.flex-h {
+  flex-direction: row;
+}
+
+.page-top {
+  height: 530px;
+  background-image: url(../images/your_verse_hero.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 0 100%;
+  padding-top: 70px;
+  flex-basis: 100%;
+}
+
+.promos {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.promos a {
+  width: 25%;
+}
```

Comment:
It will be more readable if you use meaningful comments. like (/* Start of header*/ , /* End of Hero portion */ )
 It will be helping you in the future when you want to modify or change your codes.